src/libs/stackdriverReporter.js

```diff
@@ -2,6 +2,7 @@ import { Storage } from './storage';
 import StackdriverErrorReporter from 'stackdriver-errors-js';
 import { Config } from './config';
 import * as ld from 'lodash';
+import isNil from 'lodash/fp';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6956759)
```suggestion
import isNil from "lodash/fp";
```